app/assets/stylesheets/welcome.scss

```diff
@@ -20,8 +31,15 @@ video {
    background-size: cover;
 }
 
+.hidden {
+  display: none;
+}
+.visuallyhidden {
+  opacity: 0;
+}
+
 .welcome-text {
-  margin: 400px 0 0 0;
+  margin: 200px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `200px 0 0` instead of `200px 0 0 0`